pages/styles/index.css

```diff
@@ -0,0 +1,82 @@
+* {
+    font-family: sans-serif;
+}
+
+.links {
+    position: relative;
+    left: 430px; 
+    width: 70%;
+    height: 35vh;
+    background: white;
+    font-size: 25px;
+} 
+
+nav {
+    display: flex;
+    width: 100%;
+    padding: 30px 0;
+    align-items: center;
+    justify-content: space-between;
+} 
+
+header a {
+    margin-left: 640px;
+}
+
+ul li {
+    display: inline-block;
+    list-style: none;
+}
+
+ul, li, a {
+    text-decoration: none;
+    color: #c6fbf9;
+    font-weight: bold;
+}  
+
+ul, li, a:hover{
+    color: #fbd7c6;
+}
+
+.thao {
+    margin-left: 6%;
+    margin-top: 13%;
+}
+
+.thao h1 {
+    font-size: 90px;
+    color: #c6d0fb;
+    margin-bottom: 10px;
+}
+
+span {
+    color: #c6fbce;
+    line-height: 22px;
+}
+
+p {
+    color: #f7c6fb;
+    font-size: 20px;
+}
+/* PORTFOLIO BUTTON */
+.thao a{
+    background: #c6fbf9;
+    padding: 15px 15px;
+    text-decoration: none;
+    font-weight: bold;
+    font-size: 20px;
+    color: #faffdb;
+    display: inline-block;
+    margin: 10px 5px;
+    border-radius: 3px;
+}
+
+.thao a:hover{
+    color: #ffebe0;
+}
+
+h4 {
+    color: rgb(188, 206, 251);
```

Comment:
Most of the places you have colors specified in your files you use hex notation. Here and for a few rules in about.css you instead use this `rgb` notation. And in other places you use named colors. This is all fine, but I recommend picking one way of specifying it and staying consistent.

Maintaining consistency in your code may feel minor, but inconsistencies tend to be where errors and harder to read code crops up, so I highly recommend it.